.github/workflows/deploy_improvado_service_helm_chart.yaml

```
@@ -0,0 +1,154 @@
+name: Deploy Helm chart to cluster
+on:
+  workflow_call:
+    inputs:
+      # Helm chart parameters
+      chart_name:
+        description: "Helm chart name e.g. <helm-uls-component>"
+        type: string
+        required: true
+      chart_release:
+        description: "Helm chart release name"
+        type: string
+        required: true
+      chart_version:
+        description: "Helm chart package version"
+        type: string
+        required: true
+      namespace:
+        description: "Namespace for installing helm chart"
+        type: string
+        required: true
+      values_path:
+        description: "Path to values.yaml in repo"
+        required: true
+        type: string
+      # Image parameters (docker image)
+      image_repository:
+        description: "image repository"
+        type: string
+        required: true
+      image_tag:
+        description: "image tag"
+        type: string
+        required: true
+      image_tag_path:
+        description: "Image tag path into values.yaml"
+        type: string
+        default: "image.tag"
+        required: false
+      # AWS parameters
+      aws_account_id:
+        description: "AWS account id for ECR"
+        required: true
+        type: string
+      aws_region:
+        description: "AWS region for ECR"
+        required: true
+        type: string
+      cluster_name:
+        description: "Name of AWS EKS cluster"
+        type: string
+        required: true
+      # BUILD and PUSH step
+      build_and_push_step_env:
+        description: |
+          Environment variables in json string format
+          # example:
+          build_and_push_step_env |
+            {
+              "ECR_REGISTRY": "716309063777.dkr.ecr.us-east-1.amazonaws.com",
+              "ECR_REPO": "ch-file-writer",
+              "IMAGE_TAG": "master-${{ github.run_number }}-${{ github.run_attempt }}"
+            }
+        type: string
+        required: false
+      build_and_push_step_run:
+        description: "Step for build and push image custom for many repos"
+        type: string
+        required: false
+        default: |
+          IMAGE_FULL_NAME="${ECR_REGISTRY}/${ECR_REPO}:${IMAGE_TAG}"
+          PYTHON_TAG=$(grep --max-count 1 -P '^ARG PYTHON_IMAGE=python:.+$' ./Dockerfile | cut -d "=" -f 2)
```

Назвать бы эту штуку по-другому, потому что мы по факту тут не тег получаем, а всё название образа
```suggestion
          PYTHON_IMAGE=$(grep --max-count 1 -P '^ARG PYTHON_IMAGE=python:.+$' ./Dockerfile | cut -d "=" -f 2)
```

---

Не, все же это tag имаджа, имадж дальше 
```
--build-arg PYTHON_IMAGE=harbor.tools.improvado.io/hub-cache/library/${PYTHON_TAG}
```

---

Тогда вот тут будет некрасиво. Я бы оставил PYTHON_TAG
```
--build-arg PYTHON_IMAGE=harbor.tools.improvado.io/hub-cache/library/${PYTHON_TAG}
```

---

Да наоборот красивее станет. Иначе тут ожидаешь увидеть что то вроде 
`PYTHON_IMAGE=harbor.tools.improvado.io/hub-cache/library/3.7-alpine`